nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ConfigureProxy.java

```diff
@@ -76,8 +74,8 @@ public void execute() throws BuildException {
                 getProject().setUserProperty(hostProperty, "");
                 getProject().setUserProperty(portProperty, "80");
             }
-        } catch (IOException ex) {
-            throw new BuildException(ex);
+        } catch (IOException exception) {
```

Comment:
Changes in this file could be avoided. Btw. I am removing the [unecessary import](https://github.com/apache/netbeans/pull/2225/commits/40584ecfeb52cfd7f8f5bc619ad97ad968303819) too.